.mergify.yml

```diff
@@ -72,6 +72,7 @@ queue_rules:
               - "status-success=golangci-lint"
               - "status-success=mod-check"
               - "status-success=lint-extras"
+              - "status-success=uncommitted-code-check"
```

Comment:
I see that we are backporting this to release-v3.11 branch, can you please update the mergify rules for the same?

Reply:
I don't see a rule for base branch `release-v3.11` it is only for all release branches or `v3.10` individually.

Reply:
https://github.com/ceph/ceph-csi/blob/devel/.mergify.yml#L30, https://github.com/ceph/ceph-csi/blob/devel/.mergify.yml#L177 and https://github.com/ceph/ceph-csi/blob/devel/.mergify.yml#L265, This is applicable for both 3.11 and next 3.12

Reply:
If I add it under `base~=^(release-.+)$` check, and we only backport to `release-v3.11` branch won't that cause the CI in other older branches to fail?

Reply:
can you please do it in above 3 places i pointed out?

Reply:
Sorry, missed the other places.
Added them now.